Can you fix the CI failure?
```
08:59:55 --- stderr: pcl_conversions
08:59:55 CMake Error at CMakeLists.txt:33 (target_include_directories):
08:59:55   target_include_directories may only set INTERFACE properties on INTERFACE
08:59:55   targets
```

--- a/pcl_conversions/CMakeLists.txt
+++ b/pcl_conversions/CMakeLists.txt
@@ -34,7 +34,6 @@ target_include_directories(
   ${PROJECT_NAME}
   INTERFACE
   "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>"
-  ${PCL_COMMON_INCLUDE_DIRS}
 )
 
 target_link_libraries(

--- a/pcl_conversions/CMakeLists.txt
+++ b/pcl_conversions/CMakeLists.txt
@@ -32,7 +32,7 @@ add_library(${PROJECT_NAME} INTERFACE)
 
 target_include_directories(
   ${PROJECT_NAME}
-  INTERFACE PUBLIC
+  INTERFACE
   "$<INSTALL_INTERFACE:include>"
   ${PCL_COMMON_INCLUDE_DIRS}
 )

--- a/pcl_conversions/CMakeLists.txt
+++ b/pcl_conversions/CMakeLists.txt
@@ -33,7 +33,7 @@ add_library(${PROJECT_NAME} INTERFACE)
 target_include_directories(
   ${PROJECT_NAME}
   INTERFACE
-  "$<INSTALL_INTERFACE:include>"
+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>"
   ${PCL_COMMON_INCLUDE_DIRS}
 )
 

